### ISSUE TITLE
ethminer 0.18.0-rc.0 hangs after generating DAG

### ISSUE THREAD

**Issue by geofurb** [ethereum-mining/ethminer#1862]
**Describe the bug**
When running ethminer, the entire system freezes almost immediately after ethminer prints to the console that it generated the DAG.

**To Reproduce**
1. `ethminer.exe -G -P stratum1+ssl://...`
2. Wait

**Expected behavior**
Expected typical mining

**Screenshots (Optional)**
[Picture of the screen](https://i.imgur.com/dxa09Zw.jpg), since a screencap isn't possible; the system hangs.

**Environment (please complete the following information):**
- Operating System: `Windows 10`; version 1809; build 17763.253
- Hardware `AMD Vega 56` driver version 19.2.2
- Ethminer Version `0.18.0-rc.0`
- Ethminer `-G -P`

**Comment by geofurb** [comment]
Updating `AMD Vega 56` driver to version 19.2.3 (latest) does not resolve the issue.  
Rolling back to `ethminer 0.17.1` resolves the issue.

**Comment by mrAbS777** [comment]
I have the same problem. Vega 64. Changing kernel with --cl-nobin parameter solving issue. But with opencl hashrate near 33Mh/s.  When  using 1.6.1 - 43 Mh/s with opencl kernel.

**Comment by geofurb** [comment]
Sounds like an issue with the gfx900 kernel.

**Comment by mrAbS777** [comment]
Definitely. And similar problem on Ubuntu 18.04.  It's not hang but doing 0.0 Mh/s still loading GPU. 
I've tried to compile binary  kernel for 0.16.1 which working nice on Cl-kernel on Vega64. It's missing in distribution files. But got the same problem. So the problem in binary kernel for gfx900.
